ard_controller/ard_controller.ino

```diff
@@ -278,21 +388,57 @@ void sensor_data_response() {
 }
 
 void setup() {
-  Serial.begin(115200);
-  Serial1.begin(115200);
+  Serial.begin(115200); // console
```

Comment:
can we name them better just for easy readability? serial_raspi or something?

Reply:
Sure